apps/mark/backend/src/store.ts

```diff
@@ -24,6 +29,9 @@ import { join } from 'node:path';
 
 const SchemaPath = join(__dirname, '../schema.sql');
 
+// [TODO] Fits better in ./types.ts, but this avoids a dependency cycle for now:
```

Comment:
Kind of surprising that `types.ts` depends on `store.ts`. Is the goal to move all types to `types.ts` so that it imports nothing and everything imports `types.ts`?

Reply:
> so that it imports nothing

+1 - that feels like a good rule of thumb to adopt. If importing any internal types into `types.ts`, they should be moved there instead